topo/node/cisco/cisco.go

```diff
@@ -405,6 +423,13 @@ func defaults(pb *tpb.Node) (*tpb.Node, error) {
 		if pb.Config.Image == "" {
 			pb.Config.Image = "xrd:latest"
 		}
+		if len(pb.HostConstraints) == 0 {
```

Comment:
i think checking for nil is better here in case it is unspecified

Reply:
Addressed.